app/src/main/java/vergecurrency/vergewallet/view/ui/activity/settings/DonateActivity.java

```diff
@@ -56,9 +56,9 @@ private void initComponents() {
 			FragmentSend fs = new FragmentSend(getText(R.string.donate_donation_address).toString());
 
 			FragmentManager fragmentManager = getSupportFragmentManager();
-			fragmentManager.beginTransaction()
+			/*fragmentManager.beginTransaction()
 					.replace(R.id.donate_content, fs)
-					.commit();
+					.commit();*/
```

Comment:
Is there any reason to leave dead code in here?

Reply:
@2lazy2debug @marpme  i dont know because there is no view for R.id.donate_content. Just uncommented it because of  causing build errors.

Reply:
I haven't pulled your code yet but I can ensure you R.id.donate_content is a valid FrameLayout defined in  the XML file of the donate layout (R.layout.activity_donate_layout)

Reply:
something went wrong in IDE

Reply:
It's always IDE's fault. On a more serious note, can you clean that up please?